invenio_vocabularies/contrib/subjects/schema.py

```diff
@@ -24,6 +33,16 @@ class SubjectSchema(BaseVocabularySchema):
     id = SanitizedUnicode(required=True)
     scheme = SanitizedUnicode(required=True)
     subject = SanitizedUnicode(required=True)
+    title = i18n_strings
+    synonyms = fields.List(SanitizedUnicode())
+
+    @pre_load
+    def add_subject_from_title(self, data, **kwargs):
+        """Add subject from title if not present."""
+        locale = current_app.config.get("BABEL_DEFAULT_LOCALE", "en")
```

Comment:
```suggestion
        locale = get_locale()
```

minor: this is more reliable in terms of how to fetch the default locale. If this fails for app/request context reasons, it should be fixed upstream so that it doesn't.